src/RuleFilterParser/Api/ApiRequest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace RuleFilterParser;
+
+public class ApiRequest : IApiRequest
+{
+    [FromQuery(Name = "fields")]
```

Comment:
We can mark parameters as a case insensitive.

Reply:
Understood. The reason for using `FromQuery` was to keep consistency with other Core APIs. Swagger gen tools generates a schema with PascalCase for property names. I don't have a strong opinion about it and we can leave it as it is.